From e2b2876318611a8e99378e26630348492551c959 Mon Sep 17 00:00:00 2001
From: Neal Gompa <neal@gompa.dev>
Date: Tue, 12 Sep 2023 18:25:34 -0400
Subject: [PATCH] cmake: Drop Werror from compile options

This fixes the build for the library bindings and tests.

References:
* https://github.com/rpm-software-management/dnf5/issues/746
* https://github.com/rpm-software-management/dnf5/issues/562
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 47c842e7..9fe5204e 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -89,7 +89,7 @@ set(CMAKE_VISIBILITY_INLINES_HIDDEN ON)
 add_compile_options("-fmacro-prefix-map=${PROJECT_SOURCE_DIR}/=")
 
 # warnings
-add_compile_options(-Wall -Wextra -Werror)
+add_compile_options(-Wall -Wextra)
 add_compile_options(-Wcast-align -Wformat-nonliteral -Wmissing-format-attribute -Wredundant-decls -Wsign-compare -Wsign-conversion -Wtype-limits -Wuninitialized -Wwrite-strings)
 add_compile_options(-Werror=unused-result -Wodr)
 
-- 
2.53.0

